app/views/docs/storage.phtml

```diff
@@ -0,0 +1,83 @@
+
+<h4>More Bucket Configurations</h4>
+
+<p>Unlike Collections, storage buckets have more configuration options regarding type and size of files that can be in the bucket, encryption and anti-virus scanning. If you look at the bucket settings or the REST API example above you can find these configurations. Let's look at what those are:</p>
+
+<ul>
+    <li> <code>allowedFileExtensions</code> parameter allows you to provide a list of file extensions that will be accepted into the bucket.</li>
+    <li><code>maximumFileSize</code> parameter allows you to set the maximum size of a file that a bucket accepts. The buckets will accept any file less than or equal to the size provided here.</li>
+    <li><code>encryption</code> parameter allows you to configure whether or not the files inside the bucket will be encrypted.</li>
+    <li><code>antivirus</code> parameter allows you to configure whether or not the files being added inside the bucket be scanned by antivirus.</li>
+</ul>
+
+<p>You can learn more about storage buckets and APIs you can use to manage storage buckets from our <a href="docs/server/storage">storage documentation</a>. Now that we know how to create and configure buckets for our applications, we will look at how we can manage files using the storage service next.</p>
+
+<h2>Create File</h2>
+
+<p>Once you create a bucket or have navigated to bucket details, you can access the Files tab from where you can upload, view, delete, update files in the bucket using Appwrite project's dashboard. You can perform all those operations from Appwrite's client and server SDKs as well as REST APIs with proper permission.</p>
+
+<p>Once you are in the files tab, you can click <b>Add File</b> and select a file to upload. If the bucket is configured to accept the file type and size you are uploading, your file should be uploaded and you should be able to view the file in the list of files.</p>
+
+<p>You can also upload files using REST API or SDKs. Below is a CURL syntax to create a file.</p>
+
+<!-- Create file code -->
+
+<h3>Large Files (files above 5MB)</h3>
+
+<p>Above method will only work for files up to 5MB size. If any files are above 5MB, you will need to upload them in chunks. But don't worry, if you are using Appwrite's console or any of our SDKs we handle that internally, so no matter what file size you try to upload, it will work like it worked before. Below is a code example in Dart of how you can chunk and upload each chunk. We are using the content-range header provided in the web spec in order to support chunked upload. So if you are using REST API you need to provide chunks and those headers to make a successful upload.</p>
```

Comment:
```suggestion
<p>Above method will only work for files up to 5MB in size. If any files are above 5MB, you will need to upload them in chunks. Don't worry, if you are using Appwrite's console or any of our SDKs we handle that internally, so no matter what file size you try to upload, it will work. Below is a code example in Dart of how you can chunk and upload each chunk. We are using the content-range header provided in the web spec in order to support chunked upload. So if you are using REST API you need to provide chunks and those headers to make a successful upload.</p>
```

Let's drop "as before", because in 5 years people will have no idea what was before